.github/workflows/test.yml

```diff
@@ -394,8 +307,24 @@ jobs:
       - name: Install Python packages
         uses: ./src/DDS-Router/.github/actions/install-python-packages
 
-      - name: Fetch Fast DDS repositories
-        uses: ./src/DDS-Router/.github/actions/fetch-fastdds-repos
+      - name: Get build eProsima dependencies Job Id
+        run: |
+          export JOB_ID=$(curl -sL $GITHUB_API_URL/repos/$GITHUB_REPOSITORY/actions/workflows/build_fastdds.yml/runs | jq -r '.workflow_runs[0].id?')
+          echo "fastdds_job_id=${JOB_ID}" >> $GITHUB_ENV
+
+      - name: Download eProsima dependencies
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: build_fastdds.yml
+          path: /home/runner/work/fastdds/install
+          name: ubuntu_fastdds_install
+          run_id: ${{ env.fastdds_job_id }}
+
+      - name: Install eProsima dependencies
+        run: |
+          sudo cp -r /home/runner/work/fastdds/install/foonathan_memory_vendor/* /usr/local
+          sudo cp -r /home/runner/work/fastdds/install/fastcdr/* /usr/local
+          sudo cp -r /home/runner/work/fastdds/install/fastrtps/* /usr/local
```

Comment:
line 339: source is missing: probably cp is needed because of that.
Equal in clang job